.github/workflows/opex_api_session_manager.yaml

```diff
@@ -0,0 +1,105 @@
+name: Opex Dashboard session-manager
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "apps/session-manager/api/*.yaml"
+      - ".github/workflows/opex_api_session_manager.yaml"
+      - "apps/session-manager/.opex/**"
```

Comment:
I'd add ./.github/workflows/call_opex_api.yml too.
Moreover, what should happen if it has changed? (We should add it to filters, I suppose)

Reply:
you are right about the paths for the action trigger. but I am not sure about adding it to the filters for plan/apply operations

Reply:
What dashboard do we want to update, in case those files changed? I think all of them, since those files contains the opex dashboard action, and it can change the output updating the version.

Reply:
I introduced two new filters in https://github.com/pagopa/io-auth-n-identity-domain/pull/59/commits/0f94e8defc8a925415f71412c42b86e35d2793d1 . let me know if those are okay